schemas/2015-05-21-preview/Microsoft.DevTestLab.json

```diff
@@ -38,38 +38,16 @@
         "properties": {
           "type": "object",
           "properties": {
-            "labId": {
-              "type": "string",
-              "minLength": 1,
-              "description": "Microsoft.DevTestLab/environments - Lab ID"
-            },
-            "vms": {
-              "oneOf": [
-                {
-                  "type": "array",
-                  "items": {
-                    "type": "object",
-                    "required": [ "name", "vmTemplateName", "size" ],
-                    "properties": {
-                      "name": { "type": "string" },
-                      "vmTemplateName": { "type": "string" },
-                      "size": { "type": "string" },
-                      "userName": { "type": "string" },
-                      "password": { "type": "string" },
-                      "sshKey": { "type": "string" },
-                      "isAuthenticationWithSshKey": { "type": "boolean" }
-                    }
-                  }
-                },
-                { "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
-              ],
-              "description": "Microsoft.DevTestLab/environments - VMs"
-            }
-          },
-          "required": [ "labId", "vms" ]
+            "customImageName": { "type": "string" },
+            "size": { "type": "string" },
+            "userName": { "type": "string" },
+            "password": { "type": "string" },
+            "sshKey": { "type": "string" },
+            "isAuthenticationWithSshKey": { "type": "boolean" }
+          }
         }
       },
-      "required": [ "type", "apiVersion", "properties", "location" ]
```

Comment:
In our business logic. location is not required for virtualMachine. The location will be the same as the parent resource (lab). If ARM requires it, we should add it back.